packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerApi.ts

```diff
@@ -76,6 +76,7 @@ export class OpenId4VcIssuerApi {
       issuer.credentialConfigurationsSupported = undefined
     }
     issuer.display = options.display
+    issuer.dPoPSigningAlgValuesSupported = options.dPoPSigningAlgValuesSupported
```

Comment:
I feel like we'll have more issuer metadata properties over time that we want to add. Should we have a generic way for it?